app/admin/chapter.rb

```diff
@@ -27,6 +26,11 @@
 
   form do |f|
     f.inputs "Edit Chapter" do
+      #if user only has one role and it is leader...
+      if current_admin_user.roles.count == 1 and current_admin_user.roles.first.name == "admin"
```

Comment:
This might be nice to pull into separate methods

```ruby
# You could put this in the user class (or whatever the underlying current_admin_user model)
def one_role?
  roles.count == 1
end

def admin?
  roles.first.name == "admin"
end

# Then at this line in the code you could make is concise and rubyish
if current_admin_user.has_one_role? && current_admin_user.admin?
```